Do not install same license twice, will also fix QA warning of unexpected
install paths
===================================================================
--- src.orig/CMakeLists.txt
+++ src/CMakeLists.txt
@@ -306,7 +306,6 @@ install ( TARGETS ${CORE_RUNTIME_TARGET}
   LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT asan )
 
 # Install license
-install ( FILES ${CMAKE_CURRENT_SOURCE_DIR}/LICENSE.md DESTINATION ${CMAKE_INSTALL_DOCDIR}-asan COMPONENT asan )
 install ( FILES ${CMAKE_CURRENT_SOURCE_DIR}/LICENSE.md DESTINATION ${CMAKE_INSTALL_DOCDIR} COMPONENT binary )
 
 # Install public headers
